### PR TITLE
Extracted scheduling bc docs from Michael W.'s pr #39

### PR DESCRIPTION
**Pull request by pedrosousabarreto**
Scheduling BC doc extracted from Michael Wagener's PR #39

### CODE REVIEW

**Comment by mwmentor** [comment]
Thanks @pedrosousabarreto - this is great.  I will connect to it and start using it for my new work :-)

**Comment by mwmentor** [comment]
Thanks @pedrosousabarreto - All good.

**Review by mwmentor — APPROVED** [review]
All good, thanks.

**Review by mwmentor — APPROVED** [review]
All good, thanks.

File: docs/refarch/boundedContexts/scheduling/index.md [hunk 1]
```diff
@@ -1,23 +1,133 @@
...
+
+* After the scheduled task has been initiated, does the Scheduling BC remain responsible for tracking its progress?
+
+    * Answer: No. When the Reminder is due, it is communicated to the Client BC using the precribed method, and thereafter, the Scheduling BC responsibility for the reminder is passed back to the Client BC.
+
+* Is the Client BC or the person who scheduled a Reminder noted as the "User" by the Scheduling BC?  IOW, whose ID is stamped on the process audit trail?
+
+    * Answer: This should be determined by the Client BC, based on its action on receipt of the Reminder.
+
+[^1]: For further information please view: [Link Name](https://...)
```
Comment (mdebarros) [inline docs/refarch/boundedContexts/scheduling/index.md:133]: Should this not be removed since we do not have any `[^1]` references?

**Review by mdebarros — COMMENTED** [review]
Minor comment for your attention

File: docs/refarch/boundedContexts/scheduling/index.md (hunk 1 above)
Comment (pedrosousabarreto) [reply docs/refarch/boundedContexts/scheduling/index.md]: removed

**Review by mdebarros — APPROVED** [review]
+1

**Review by mwmentor — APPROVED** [review]
All good, thanks.

**Pull request merged**